docs/lib/dynamic_config.md

```diff
@@ -0,0 +1,579 @@
+> Dynaconf has some merging strategies that can be configured, read more on [dynaconf] documentation.
+> By default the latest set key will completely override the previous one.
+> unless it explicitly uses one of the `@merge`. "@merge_unique", "@insert" or "dynaconf_merge" directives.
+>
+> If `merge=True` is passed to the `factory` function, then dynaconf will operate on the global merging > strategy, which means lists and dictionaries will be merged back to previous by default (even without the directives) instead of replaced.
+>
+> Notice that DAB doesn't use the `merge=True` option by default, so the default behavior is to replace the previous settings, this allows more granular control over the settings merge.
+
+
+#### Check the current mode 
+
+For example `export MYAPP_MODE=production` will set the mode to production,
+modes can also be composed like `export MYAPP_MODE=production,extra`
+
+```python
+DYNACONF.is_development_mode # True if development
+"production" in DYNACONF.current_env.lower() # True if production
+```
+
+#### Loading Standard Ansible Automation Platform settings files
+
+The `load_standard_settings_files` function will load settings from the standard locations,
+the base location is standardized as `/etc/ansible-automation-platform/` and the function will load
+in order.
+
+- `/etc/ansible-automation-platform/{settings, flags, .secrets}.yaml`
+- `/etc/ansible-automation-platform/{name}/{settings, flags, .secrets}.yaml`
+
+where `{name}` is the lowercase name passed to the factory function.
+
+```python
+load_standard_settings_files(DYNACONF)
+```
+
+Those files will be loaded in order and the merging strategy will be applied.
+
+#### Load settings from environment variables
+
+The `load_envvars` function will load settings from environment variables using the standard prefix
+`{NAME}_` where `{NAME}` is the upper name passed to the factory function.
+
+If extra prefixes are needed they can be passed as a list to the function.
+
+```python
+load_envvars(DYNACONF)
+```
+
+and then the settings can be externally overridden as:
+
+```bash
+export MYAPP_KEY=value
+export MYAPP_KEY__SUBKEY=value
+export MYAPP_DATABASES__default__NAME=mydb
+export MYAPP_NUMBER=42
+export MYAPP_LIST="[1, 2, 3]"
+export MYAPP_DICT='@json {"key": "value"}'
+# or a valid toml
+export MYAPP_DICT='{key="value"}'
+export MYAPP_BOOL=true
+export MYAPP_FLOAT=3.14
+export MYAPP_INTERPOLATE="@format The key value is {this.KEY}"
+export MYAPP_INTERPOLATE_COMPLEX="@int @jinja {{ this.NUMBER * 4}}"
+export MYAPP_INSTALLED_APPS="@merge new_app_to_add"
+```
+
+More on [dynaconf] documentation.
+
+
+### Export settings back to django settings
+
+The export function will take all variables from the dynaconf object and export them back to django settings so it will be available as `django.conf.settings`:
+
+It is important to call this function at the very end of the `settings` module.
```

Comment:
If we can't auto-do this can we change this to 
```suggestion
It is important to call this function at the very end of the `settings` module but right before the `validate` call (see next section).
```

Reply:
I will make the change to not require the `validate` call.